Add stream with no funds to claim

agnieszkajarosikloj opened the pull request:
## Description

This PR adds active stream with no funds to claim. UI only. Values and functions are mocks.
After clicking "Claim funds" button:
- "Available to claim" funds  are equal to 0,
-  button displays "Claimed" text and is disabled,
-  "Ended" state is visible,
- "passed" state below linked motions is displayed.
The rest of the states (failed, cancelled, ...) will be added in other PRs.

<img width="430" alt="Screenshot 2022-09-22 at 14 53 14" src="https://user-images.githubusercontent.com/91876137/191752183-63fa3c98-16d3-4629-88a3-9e6e3412f242.png">



[Figma link](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=5544%3A601778)


Resolves #3856

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/JoinColony/colonyDapp?pullRequest=3889) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/JoinColony/colonyDapp?pullRequest=3889) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/JoinColony/colonyDapp?pullRequest=3889) it.</sub>

willm30 reviewed (commented):
Looks good. 

Couple of things: 
1. I know the data is mocked, but why is the number showing up twice?
2. The "passed" state shows up for me before I've clicked "Claimed"

![claim](https://user-images.githubusercontent.com/64402732/192990967-ad3af6d2-f35f-4f36-aacb-ed9d24981328.png)

agnieszkajarosikloj commented:
@willm30 
1. I changed mocked data, because token was incorrect. And I fixed displaying amount, when token was not found.
2. The "passed" status refers to the motion to create streaming payment (after clicking "Start stream" button).

willm30 reviewed (approved):
Nice one 👍

jakubcolony commented on src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.tsx:
```diff
@@ -288,13 +306,14 @@ const StreamingStagesLocked = ({
-                        [styles.marginBottom]:
-                          availableToClaim.length > 1 &&
-                          index !== availableToClaim.length - 1,
+                        [styles.marginTop]: index > 0,
```
Similarly to my question in #3888, wouldn't it make sense for that to be done in CSS?

chinins replied:
I agree, I think you can use css selectors for that.

jakubcolony reviewed (approved):
Looks good to me!

<img width="406" alt="image" src="https://user-images.githubusercontent.com/112586815/193274588-707a7bef-809c-412b-a128-73afec3f19f3.png">

chinins commented on src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.css:
```diff
@@ -130,6 +135,6 @@
 
-.marginBottom {
-  margin-bottom: 24px;
+.marginTop {
```
We try not to use class names like these. Please, provide a more meaningful name (e.g. container of sorts).

chinins commented on src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.tsx:
```diff
@@ -57,6 +57,14 @@ const MSG = defineMessages({
+    defaultMessage: 'Ended',
+  },
+  claimed: {
+    id: `dashboard.ExpenditurePage.Stages.StreamingStage.StreamingStagesLocked.ended`,
```
wrong id - should be `claimed` at the end

chinins reviewed (commented):
All good, just please update the css as per Jakub's and my comment. Please, let me know once done & i'll re-review straight away.
